shared/src/common/types/billing/billing-card.type.ts

```diff
@@ -0,0 +1,25 @@
+type BillingCard = {
+  id: string;
+  object: string;
```

Comment:
If that's a duplicate type from stripe's ts declaration, then it seems like object is of `card` type